python/lsst/fgcmcal/fgcmBuildStarsBase.py

```diff
@@ -516,15 +516,15 @@ def _fillVisitCatalog(self, visitCat, groupedDataRefs, bkgDataRefDict=None,
 
         Parameters
         ----------
-        visitCat: `afw.table.BaseCatalog`
-           Catalog with schema from _makeFgcmVisitSchema()
-        groupedDataRefs: `dict`
-           Dictionary with visit keys, and `list`s of
-           `lsst.daf.persistence.ButlerDataRef`
-        visitCatDataRef: `lsst.daf.persistence.ButlerDataRef`, optional
-           Dataref to write visitCat for checkpoints
-        bkgDataRefDict: `dict`, optional
-           Dictionary of gen3 dataRefHandles for background info. FIXME
+        visitCat : `afw.table.BaseCatalog`
+            Catalog with schema from _makeFgcmVisitSchema()
+        groupedDataRefs : `dict`
+            Dictionary with visit keys, and `list`s of
+            `lsst.daf.persistence.ButlerDataRef`
+        visitCatDataRef : `lsst.daf.persistence.ButlerDataRef`, optional
```

Comment:
Can these also be whatever the Gen3 version of `lsst.daf.persistence.ButlerDataRef` is (and for `groupedDataRefs` above)?

Reply:
`groupedDataRefs` can be either (updated), but `visitCatDataRef` is Gen2 only, and a note has been added.